conf/supporting_evidence.routes

```diff
@@ -0,0 +1 @@
+GET     /supporting-evidence  uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.supportingevidence.SupportingEvidenceController.onHoldingPageLoad
```

Comment:
This end point is not correct. This is the correct one for now:`GET    /supporting-evidence/scan-progress`

Actually we need to pass a UUID but I will do that as I building that out at the moment.